providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/config.py

```diff
@@ -19,10 +20,13 @@ class CacheType(Enum):
 DEFAULT_HOST = "localhost"
 DEFAULT_KEEP_ALIVE = 0
 DEFAULT_OFFLINE_SOURCE_PATH: typing.Optional[str] = None
+DEFAULT_OFFLINE_POLL_MS = 5000
```

Comment:
i added this to be compliant with the config gherkin test - usage of this will be fixed as soon as we touch the in-process provider